sources/packages/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { Controller, Get, Injectable, NotFoundException } from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { IUserToken } from "../../auth/userToken.interface";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, UserToken } from "../../auth/decorators";
+import { StudentFileService, StudentService } from "../../services";
+import BaseController from "../BaseController";
+import { StudentUploadFileDTO } from "./models/student.dto";
+import { ClientTypeBaseRoute } from "src/types";
+
+/**
+ * Student controller for Student Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.student)
+@Controller("students")
+@ApiTags(`${ClientTypeBaseRoute.Student}-students`)
+@Injectable()
+export class StudentStudentsController extends BaseController {
+  constructor(
+    private readonly studentService: StudentService,
+    private readonly fileService: StudentFileService,
+  ) {
+    super();
+  }
+
+  /**
+   * This controller returns all student documents uploaded
+   * by student uploader.
+   * @returns list of student documents.
+   */
+  @Get("documents")
+  @ApiNotFoundResponse({ description: "Student not found." })
```

Comment:
Can we improve it to be clear that there is no student account created to the user, like:
```
@ApiNotFoundResponse({ description: "The user does not have a student account associated with." })
```